tasks/section5.yml

```diff
@@ -111,15 +127,19 @@
       - rule_5.1.6
 
 - name: "SCORED | 5.1.6 | PATCH | Ensure permissions on /etc/cron.monthly are configured"
-  command: /bin/true
+  file:
+      dest=/etc/cron.monthly
+      owner=root
+      group=root
+      mode=0600
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md